src/components/Login/Login.module.css

```diff
@@ -0,0 +1,43 @@
+@import url('../../theme.module.css');
```

Comment:
Variables are added to the root so I'm wondering if we need this import?

Reply:
Didnt think on it, good point!